src/flask/cli.py

```diff
@@ -298,22 +298,20 @@ def load_app(self) -> Flask:
         if self._loaded_app is not None:
             return self._loaded_app
 
-        if self.create_app is not None:
+    if self.create_app is not None:
```

Comment:
Correct the code indentation. The code doesn't run.

Reply:
@JeremieBergeron just fixed code indentation, thank you for the remark, should I merge it like this ?

Reply:
@jaykay9999 Could you say if we need to had an entry in ``CHANGES.rst`` for this PR like said in the [contributing guide](https://github.com/jaykay9999/flaskreviewthree/blob/main/CONTRIBUTING.rst#submitting-patches)?
The docs says we should, but it seems overkill add entry for a small refactor like this one.

Reply:
@JeremieBergeron you're correct, for a change have an entry to CHANGES.rst  we must also include .. versionchanged:: which might be too much for a small change. Do I proceed to merge the code like this ?

Reply:
Yes, you can merge.